topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -318,21 +462,21 @@ To accomplish this task we will use [Cutadapt](https://cutadapt.readthedocs.io/e
 >          {: .tip}
 >
 >       - In *Read 1 Options*
->
->          In this training, no adapters were already removed. If you know which adapter sequences were used during library preparation, provide their sequences there.
->
->    - In *"Adapter Options"*
->       - *"Minimum overlap length"*: `3`
+>          
+>          > ### {% icon comment %} Know adapters
+>          > In this dataset, no adapters were found as we saw in FastQC report. They were already removed. 
+>          > If you see or know which adapter sequences were used during library preparation, provide their sequences there.
```

Comment:
Would be interesting to have a data set with adapters. With different adapters for forward and reverse strand we often found that using the common part is sufficient and maybe even better.

Reply:
We could have a dataset with adapters as an extra exercise. Do you one dataset with some?

Reply:
unfortunately not.

Reply:
good idea, I think I might be able to find us a good dataset, will look into it (but let's not wait for it here)